test/http_to_do_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule HTTPServerTest.ToDo do
+  alias ToDo.API
+  use ExUnit.Case, async: true
+  doctest HTTPServer
+
+  setup do
+    mock_data = JSON.encode!(%{"todo1" => "Act", "todo2" => "Arrange"})
+    File.write!("lib/http_server_test_fixture/mock_data/test-data.json", mock_data)
```

Comment:
Is the `:file_path` here something the test should know about? or should it belong in environment configuration? (kind of like my other question above)

Reply:
alternatively, do you even need a file for the tests to write to? Its not your job test another library, should this mock just be a spy mock and watch that a call to a function was made?

Reply:
I had been having some difficulties using the `:file_path` in the tests. But I think that's because I was trying to add the `:file_path` to the test context. I went back and added this to the top of the file:
```
@file_path Application.compile_env(:http_server, :file_path, "lib/http_server_test_fixture/mock_data/test-data.json")
```
and used @file_path instead of the file path string and my tests are passing still.